### ISSUE TITLE
PHP warnings appearing in Log

### ISSUE THREAD

**Issue by sbgraphicdesign** [backdrop-contrib/entity_plus#8]
I'm getting php warnings in my log:

`Warning: in_array() expects parameter 2 to be array, boolean given in entity_plus_entity_info_alter() (line 199 of /public_html/modules/entity_plus/entity_plus.module).`

`Warning: class_implements(): Class BackdropDefaultEntityController does not exist and could not be loaded in entity_plus_entity_info_alter() (line 199 of /public_html/modules/entity_plus/entity_plus.module).`

These two notices appear together and only when I'm logged in as admin. Not sure if this is related to another module, but appears to occur on every page I visit.

**Comment by herbdool** [comment]
I think it's another module that might rely on entity_plus. I don't see the `BackdropDefaultEntityController` class in the text of this module (other than in comments, which is incorrect but won't throw an error). Any reference to that class should be just `DefaultEntityController`.

**Comment by sbgraphicdesign** [comment]
Hmm, I uninstalled Entity Reference module because I was having issues with the autocomplete. And coincidentally those errors appear to have stopped, yet there's no reference to `BackdropDefaultEntityController` that I could find in the Entity Reference module.

I was also making changes to my node templates (adding `if (isset` to call the fields only if there's a value), so not sure if this perhaps fixed the issue. Dunno how all this stuff works...

Incidently I found a reference in the XML Sitemap Menu sub module, in the "xmlsitemap_menu.module" file, on line 26: `'controller class' => 'BackdropDefaultEntityController',`, but this module is still switched on and I'm not seeing any errors.

Anyway it appears I'm no longer troubled by this and if anyone else encounters it perhaps some of the above might help them!
